torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1843,8 +1843,9 @@ void InitXlaModuleBindings(py::module m) {
                                   py::call_guard<py::gil_scoped_release>());
   m.def("_xla_get_distributed_runtime_service",
         [](int num_nodes) -> std::unique_ptr<xla::DistributedRuntimeService> {
+          std::string port = runtime::sys_util::GetEnvString("COORDINATOR_PORT", "8547");
           std::string dist_service_addr =
-              runtime::sys_util::GetEnvString("PJRT_DIST_SERVICE_ADDR", "");
+              runtime::sys_util::GetEnvString("MASTER_ADDR", "127.0.0.1") + ":" + port;
```

Comment:
For better readability, can we introduce a variable to describe this IP address is the default parameter? e.g/ `LOCAL_HOST_IP_DEFAULT`

Reply:
sure

Reply:
done